composer.json

```diff
@@ -4,53 +4,61 @@
     "type": "project",
     "license": "AGPL-3.0-only",
     "require": {
-        "php": "^7.4",
+        "php": "^8.1",
         "ext-ctype": "*",
         "ext-iconv": "*",
         "ext-json": "*",
         "api-platform/core": "^2.6",
         "cloudinary/cloudinary_php": "^2.7",
-        "danskernesdigitalebibliotek/agency-auth-bundle": "^2.0",
+        "danskernesdigitalebibliotek/agency-auth-bundle": "^3.0",
         "doctrine/annotations": "^1.10",
         "doctrine/doctrine-bundle": "^2.5",
         "doctrine/doctrine-migrations-bundle": "^3.0",
         "doctrine/orm": "^2.11",
         "nelmio/cors-bundle": "^2.2",
         "phpdocumentor/reflection-docblock": "^5.3",
         "phpstan/phpdoc-parser": "^1.2",
-        "predis/predis": "^1.1",
-        "sensio/framework-extra-bundle": "^5.5",
-        "symfony/asset": "5.2.*",
-        "symfony/cache": "5.2.*",
-        "symfony/console": "5.2.*",
-        "symfony/dotenv": "5.2.*",
-        "symfony/expression-language": "5.2.*",
-        "symfony/filesystem": "5.2.*",
-        "symfony/flex": "^1.3.1",
-        "symfony/framework-bundle": "5.2.*",
-        "symfony/http-client": "5.2.*",
-        "symfony/messenger": "5.2.*",
+        "predis/predis": "^2.0",
+        "psr/cache": "^3.0",
+        "sensio/framework-extra-bundle": "^6.2",
+        "symfony/amqp-messenger": "6.1.*",
+        "symfony/asset": "6.1.*",
+        "symfony/cache": "6.1.*",
+        "symfony/console": "6.1.*",
+        "symfony/doctrine-messenger": "6.1.*",
+        "symfony/dotenv": "6.1.*",
+        "symfony/expression-language": "6.1.*",
+        "symfony/filesystem": "6.1.*",
+        "symfony/flex": "^2.2",
+        "symfony/form": "6.1.*",
+        "symfony/framework-bundle": "6.1.*",
+        "symfony/http-client": "6.1.*",
+        "symfony/messenger": "6.1.*",
         "symfony/monolog-bundle": "^3.5",
-        "symfony/property-access": "5.2.*",
-        "symfony/property-info": "5.2.*",
-        "symfony/proxy-manager-bridge": "5.2.*",
-        "symfony/security-bundle": "5.2.*",
-        "symfony/serializer": "5.2.*",
-        "symfony/twig-bundle": "5.2.*",
-        "symfony/validator": "5.2.*",
-        "symfony/yaml": "5.2.*",
-        "vich/uploader-bundle": "^1.12"
+        "symfony/property-access": "6.1.*",
+        "symfony/property-info": "6.1.*",
+        "symfony/proxy-manager-bridge": "6.1.*",
```

Comment:
Do we need symfony/proxy-manager-bridge?

Reply:
No